src/routes/Measurements/components/nettests/Dash.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+
+import { mlabServerToCountry, mlabServerToName } from '../../../../util/nettest'
+
+/*
+ * This table is derived from:
+ * https://support.google.com/youtube/answer/1722171?hl=en-GB
+ */
+
+const minimumBitrateForVideo = [
+  {
+    'sfr_min_bitrate': 600,
+    'hfr_min_bitrate': 1000,
+    'type': '240p'
+  },
+  {
+    'sfr_min_bitrate': 1000,
+    'hfr_min_bitrate': 1500,
+    'type': '360p'
+  },
+  {
+    'sfr_min_bitrate': 2500,
+    'hfr_min_bitrate': 4000,
+    'type': '480p'
+  },
+  {
+    'sfr_min_bitrate': 5000,
+    'hfr_min_bitrate': 7500,
+    'type': '720p (HD)'
+  },
+  {
+    'sfr_min_bitrate': 8000,
+    'hfr_min_bitrate': 12000,
+    'type': '1080p (full HD)'
+  },
+  {
+    'sfr_min_bitrate': 16000,
+    'hfr_min_bitrate': 24000,
+    'type': '1440p (2k)'
+  },
+  {
+    'sfr_min_bitrate': 35000,
+    'hfr_min_bitrate': 53000,
+    'type': '2160p (4k)'
+  },
+]
+
+const getOptimalVideoRate = (test_keys) => {
+  let optimalRate = minimumBitrateForVideo[0]
+
+  minimumBitrateForVideo.forEach((rate) => {
+    if (test_keys.simple.median_bitrate >= rate['sfr_min_bitrate']) {
+      optimalRate = rate
+    }
```

Comment:
Then, I am not sure I totally understand what the code does. Let me write down a simple example and see what happens. Maybe I am confused.